blob/blob.go

```diff
@@ -57,11 +62,47 @@ func (p Proof) equal(input Proof) error {
 	return nil
 }
 
+type jsonProof struct {
+	Start int      `json:"start"`
+	End   int      `json:"end"`
+	Nodes [][]byte `json:"nodes"`
+}
```

Comment:
I curious, why ignoreMaxNamespace param is not included? I think if it is added, it will remove dependancies on node setting (ipld.NMTIgnoreMaxNamespace) on unmashal caller side.

Would support for non-inclusion proof be used with marshaller in future? If so maybe add `leafhash` []byte now, so it till support non-inclusion roofs right away.

Reply:
We don't need to support Blob Non-Inclusion proofs, but only for the whole namespace. I will send you a link with the discussion about that

Reply:
Thank you for info! So please, dismiss my second question. I still don't know the usecase for marshal / unmarshal in blob, so not sure if first question is relevant.

Reply:
ignoreMaxNamespace is a protocol constant, so there is no need to pass it